front/pages/api/v1/w/[wId]/members/validate.ts

```diff
@@ -0,0 +1,58 @@
+import type { WithAPIErrorResponse } from "@dust-tt/types";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { getMembers } from "@app/lib/api/workspace";
+import { withPublicAPIAuthentication } from "@app/lib/api/wrappers";
+import type { Authenticator } from "@app/lib/auth";
+import { apiError } from "@app/logger/withlogging";
+
+export type ValidateMemberResponseBody = {
+  valid: boolean;
+};
+
+/**
+ * @ignoreswagger
+ * Validates an email corresponds to an active member in a specific workspace. For Dust managed apps only - undocumented.
+ */
+
+async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<WithAPIErrorResponse<ValidateMemberResponseBody>>,
+  auth: Authenticator
+): Promise<void> {
+  const { email } = req.body;
+
+  switch (req.method) {
+    case "POST":
+      const { members: allMembers } = await getMembers(auth, {
```

Comment:
This is incredibly costly.

let's instead.

```
UserResource.fetchByEmail
```
then 
```
MembershipResource.getActiveMembershipOfUserInWorkspace
```

Reply:
Ok - just c/ced from the members endpoint, switching